src/cli/introspect/index.ts

```diff
@@ -0,0 +1,79 @@
+import * as fs from "fs";
+import { execute, introspectionQuery, parse } from "graphql";
+import "isomorphic-fetch";
+import * as path from "path";
+import * as yargs from "yargs";
+import { ObjectMap } from "../../types";
+
+function mkdirSyncRec(filePath: string): void {
+  const parsed = path.parse(filePath);
+  const dirPaths = parsed.dir.split(path.sep);
+
+  dirPaths.reduce((existingPath, dirPath) => {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The return value of "reduce" must be used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS2201)